training/gpu_training_job.js

```diff
@@ -73,7 +73,9 @@ module.exports = class GPUTrainingJob extends BaseTrainingJob {
         try {
             await this._processResponse();
         } finally {
-            await this._scaleGPUNode(0);
+            if (this._killed !== true) {
+              await this._scaleGPUNode(0);
+            }
```

Comment:
Why this change? Even if the job was killed we still need to scale down, right?

Reply:
this._scaleGPUNode(0) iscalled when the job is killed.  This avoids calling eskctl scale twice. Although the op is idempotent, I haven't tested two concurrent eskctl scale calls.